File: aoj/volume100/java/A10001.java
import java.util.Scanner;

class A10001 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int x = scan.nextInt();

        int cube = (int)Math.pow(x, 3);
        System.out.println(cube);
    }
}

File: aoj/volume100/nodejs/A10001.js
function main(lines) {
  var n = parseInt(lines[0], 10);
  var cube = Math.pow(n, 3);
  console.log(cube);
}

var chunks = [];
process.stdin.on('data', function(chunk) {
  chunks.push(chunk)
});
process.stdin.on('end', function() {
  var lines = chunks.join('').split('\n');
  main(lines);
});
